composer.json

```
@@ -16,7 +16,7 @@
         "wordpress", "wp", "plugin"
     ],
     "require": {
-        "php": ">=7.4|^8",
+        "php": "^7.4||^8.1",
```

Ei tarvitse enää 7.x.x versiota
```suggestion
        "php": "^8.1",
```